### ISSUE TITLE
Question: Expected configuration for use with Lambda Layers

### ISSUE THREAD

**Issue by VED-StuartMorris** [AnomalyInnovations/serverless-bundle#117]
Hello,

Just a question on configuration, we use Lambda Layers for some of our dependencies. 

What is the expected configuration for using layers? 

- Should we be installing the dependencies at the service level, as well as the layer level?
-- I.e.  the node_module is installed in the ```layers/commonLib/``` path as well as the root level of the service 

- Then should it just be excluded with ```forceExclude``` for the serverless-bundle. 

The above seems to work as expected, but there doesn't seem to be any documentation confirming this is the expected configuration approach. Thanks

**Comment by jayair** [comment]
Hmm we don't really have anything specific for layers other than the `forceExclude` option. I guess it depends on what you are putting in your layers.

**Comment by VED-StuartMorris** [comment]
@jayair Thanks for coming back, we tend to put shared resources in layers. Whether that is custom npm modules or re-used npm modules across different layers.

**Comment by jayair** [comment]
Yeah maybe we need to add some documentation for layers then. If you have some suggestions, consider opening a PR!
